lib/3scale_toolbox/commands/service_command/service_create.rb

```diff
@@ -0,0 +1,33 @@
+module ThreeScaleToolbox
+  module Commands
+    module ServiceCommand
+      class ServiceCreateSubcommand < Cri::CommandRunner
+        include ThreeScaleToolbox::Command
+
+        def self.command
+          Cri::Command.define do
+            name        'create'
+            usage       'create'
+            summary     'service create'
+            description 'Create a service'
+            runner ServiceCreateSubcommand
+
+            param   :source
+            param   :service_name
```

Comment:
The command format is:
`3scale service create  [flags] <source> <service-name>`

Reply:
:+1: